Mathlib/CategoryTheory/Limits/Preserves/Finite.lean

```diff
@@ -156,8 +219,6 @@ noncomputable instance (priority := 120) PreservesColimits.preservesFiniteColimi
   PreservesColimitsOfSize.preservesFiniteColimits F
 #align category_theory.limits.preserves_colimits.preserves_finite_colimits CategoryTheory.Limits.PreservesColimits.preservesFiniteColimits
 
--- Porting note: is this unnecessary given the instance
--- `PreservesColimitsOfSize0.preservesFiniteColimits`?
```

Comment:
This porting note was based on the misunderstanding that `PreservesColimitsOfSize0.preservesFiniteColimits` and `preservesFiniteColimitsOfPreservesFiniteColimitsOfSize` have anything to do with each other.